### ISSUE TITLE
improve site accessibility

### ISSUE THREAD

**Issue by YazeedAlKhalaf** [GDSC-YU/tuwaiq-classrooms#11]
site accessibility should be improvied using `aria-*` tags.
this makes our site easy to use for people who have impairments.
